scripts/constAddressDeployer.js

```diff
@@ -46,7 +46,7 @@ const deployContractConstant = async (
   contractJson,
   key,
   args = [],
-  gasLimit = null,
```

Comment:
there are external dependencies on this. can you add txOptions as a separate param and use that if gasLimit isn't provided?

Reply:
Instead of a separate param we check if it is a number and treat it as the gasLimit, otherwise as the full options